packages/wrangler/src/pages.tsx

```diff
@@ -1581,8 +1583,10 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
           watch,
           plugin,
         }) => {
-          // Beta message for `wrangler pages <commands>` usage
-          logger.log(pagesBetaWarning);
+          if (!isInPagesCI) {
+            // Beta message for `wrangler pages <commands>` usage
+            logger.log(pagesBetaWarning);
```

Comment:
is this the only one to silence? I see 11 instances in the codebase, but just confirming they they won't be hit in CI.

Reply:
Yes, this is the only command that _we_ run in CI. I figure if users are running other commands themselves in CI, we still want to warn them.